climpred_skeleton/core.py

```diff
@@ -0,0 +1,28 @@
+import xarray as xr
+
+
+class Verification:
```

Comment:
Verification sounds like the end product, but this seems rather to be a class the just holds two xr objects. I'd rather name this 'base', 'PredictionObservation','input_xr_objects'.

Reply:
I like `Verification` since everything that inherits this is part of a verification pipeline. So they are inheriting features from `Verification`. This will hold all the useful attributes (like `_nmembers` and methods that can be used for verification.

`Base` seems vague. Maybe `BasePrediction`. But I still like `Verification`. Let me know your thoughts

Reply:
Or maybe `CorePrediction`

Reply:
I think I am fine with Verification. I misunderstood that this not only holds the two objects but is inherited upwards so some kind of end product. sorry for the incorrect wording here. but its definately more than a 2 xr.object container.

Reply:
ok with `verification`